familyconnect_registration/settings.py

```diff
@@ -243,3 +250,19 @@
     'VHT_PUBLIC_REGISTRATION_NOTIFICATION_TEXT', "There is a new pregnancy in "
     "your parish. Call {mother} and visit the mother to update her "
     "registration.")
+LOCATION_UPDATE_REMINDER_TEXT_ENG_UG = os.environ.get(
+    'LOCATION_UPDATE_REMINDER_TEXT_ENG_UG', 'To make sure you can receive '
```

Comment:
The ones above this are `UG_ENG` rather than `ENG_UG` I'm not sure if it matters?
Why do you need one for each different language? Is this how the translations are handled?